UserInterface.adoc

```diff
@@ -1,44 +1,92 @@
-=== User interface conventions
+This section aims to introduce the user to Zyn's UI elements and explain their behaviour in detail.
+
+// TODO: Virtual keyboard panel
+
 === Panels
+
 When you run Zyn-Fusion you'll be presented with this window:
 
+// TODO: Color the elements here and use the colors as help.
```

Comment:
I think one of the plans with the automated screen capture wasn't recoloring, but making it semi-easy to draw bounding boxes around areas you wanted to highlight to users. I do not recall the status of that functionality however.

Reply:
Understandable, thank you for letting me know! I won't force such a functionality, then.